manifests/d/Diskuv/opam/2.2.0-alpha0-20221104/Diskuv.opam.installer.yaml

```diff
@@ -0,0 +1,47 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Follow:
+#   https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Cversion-example#tips-and-best-practices
+#   https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0#best-practices
+# Reference: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.2.0/installer.md
+PackageIdentifier: Diskuv.opam
+PackageVersion: 2.2.0-alpha0-20221104 # BUMP
+# Scope: user
+UpgradeBehavior: install
+Commands:
+  - opam
+  - opam-installer
+  - opam-putenv
+Platform:
+  - "Windows.Desktop"
+# Minimum OS comes from https://github.com/diskuv/dkml-runtime-distribution/blob/ff7e6bcbf3dbdb37e64b20a57e2dea0fa9d27ed5/src/windows/Machine/Machine.psm1#L14:
+# > $Windows10SdkVer = "18362"
+MinimumOSVersion: 10.0.18362.0
+InstallerType: exe
+InstallModes:
+  - silent
+  - interactive
+InstallerSwitches:
+  Silent: --ci --quiet --color=never
+  SilentWithProgress: --ci
+  Interactive: --color=always
+  InstallLocation: --prefix "<INSTALLPATH>"
+AppsAndFeaturesEntries:
+  # Keep in sync with installer\src\private_common.ml which is used by
+  # https://github.com/diskuv/dkml-install-api/blob/main/package/console/common/Windows_registry.ml
+  - Publisher: Diskuv, Inc.
+    DisplayName: Diskuv OCaml
+    DisplayVersion: 2.2.0~alpha0~20221104 # BUMP
```

Comment:
```suggestion
```

The version of the package matches Control Panel, so this DisplayVersion is incorrect and is not needed.